ts/create-kpt-functions/tsconfig.json

```diff
@@ -1,21 +1,27 @@
 {
   "compilerOptions": {
-    "module": "commonjs",
-    "target": "es6",
-    "resolveJsonModule": true,
+    "allowUnreachableCode": false,
+    "allowUnusedLabels": false,
+    "baseUrl": ".",
+    "declaration": true,
     "esModuleInterop": true,
-    "strict": true,
     "forceConsistentCasingInFileNames": true,
-    "noUnusedLocals": true,
+    "lib": ["es2018"],
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "noEmitOnError": true,
+    "noFallthroughCasesInSwitch": true,
     "noImplicitAny": true,
     "noImplicitReturns": true,
-    "moduleResolution": "node",
-    "sourceMap": true,
+    "noUnusedLocals": true,
     "outDir": "dist",
-    "baseUrl": "..",
-    "lib": ["es2015"],
-    "rootDir": "src/"
+    "pretty": true,
+    "resolveJsonModule": true,
+    "rootDir": ".",
```

Comment:
Why is this changing from 'src/' to '.'.

What's the implication? Generally tsconfig.json fields are important, so need to understand what they do if you're changing them.